package.json

```diff
@@ -24,21 +28,45 @@
     "container",
     "Google Container Engine API"
   ],
+  "contributors": [
+    "Alexander Fenster <fenster@google.com>",
+    "Ethan Bao <cbao@google.com>",
+    "Google API Publisher <googleapis-publisher@google.com>"
+  ],
+  "scripts": {
+    "publish-module": "node ../../scripts/publish.js container",
```

Comment:
This is an old thing we used in the mega-repo. It has no function anymore, since publishing happens through the GH Releases UI.

Reply:
ok, this one needs to be removed here and in future packages as well. I will take care of that.